controllers/userController.js

```diff
@@ -0,0 +1,70 @@
+import * as dotenv from "dotenv";
+import bcrypt from "bcrypt";
+import User from "../model/User.js";
+import jwt from "jsonwebtoken";
+
+export const handleSignup_simple = async (req, res) => {
+  try {
+    const hashedPwd = await bcrypt.hash(req.user.password, 10);
+    const result = await User.create({
+      names: req.user.names,
+      email: req.user.email,
+      password: hashedPwd,
```

Comment:
don't return the password in results and do this to all results with password

Reply:
Okay